sdk/src/rest/commands/update/extensions.ts

```diff
@@ -0,0 +1,27 @@
+import type { ApiOutput as Extension } from '@directus/extensions';
+import type { NestedPartial } from '../../../types/index.js';
+import type { RestCommand } from '../../types.js';
+import { throwIfEmpty } from '../../utils/index.js';
+
+/**
+ * Update an existing extension.
+ * @param bundle - Bundle this extension is in
+ * @param name - Unique name of the extension
+ * @returns Returns the extension that was updated
+ */
+export const updateRelation =
+	<Schema extends object>(
+		bundle: string | null,
+		name: string,
+		data: NestedPartial<Extension>
+	): RestCommand<Extension, Schema> =>
+	() => {
+		throwIfEmpty(name, 'Name cannot be empty');
```

Comment:
I think it's more important to check `bundle` for emptiness in case it's not `null`, so we don't end up with `/extensions//{name}`.

I think we can omit it for `name` 👍